Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -326,31 +334,51 @@ else if (e.getSeverity() == CqlTranslatorException.ErrorSeverity.Info) {
         }
     }
 
+    private boolean shouldReport(CqlTranslatorException.ErrorSeverity errorSeverity) {
+        switch (errorLevel) {
+            case Info:
+                return
+                        errorSeverity == CqlTranslatorException.ErrorSeverity.Info
+                                || errorSeverity == CqlTranslatorException.ErrorSeverity.Warning
+                                || errorSeverity == CqlTranslatorException.ErrorSeverity.Error;
+            case Warning:
+                return
+                        errorSeverity == CqlTranslatorException.ErrorSeverity.Warning
+                                || errorSeverity == CqlTranslatorException.ErrorSeverity.Error;
+            case Error:
+                return errorSeverity == CqlTranslatorException.ErrorSeverity.Error;
+            default:
+                throw new IllegalArgumentException(String.format("Unknown error severity %s", errorSeverity.toString()));
+        }
+    }
```

Comment:
I know that cql-to-elm doesn't necessarily have to be performant, and hopefully errors don't happen too often, but another approach here would be to do the comparisons based on the enum's ordinal values.

In other words, I think most of the above could be replaced by:
```java
private boolean shouldReport(CqlTranslatorException.ErrorSeverity errorSeverity) {
  return errorSeverity.ordinal() >= errorLevel.ordinal();
}
```

See: https://stackoverflow.com/questions/11550118/how-to-get-enums-numeric-value